incident_test.go

```diff
@@ -341,3 +341,38 @@ func TestIncident_ListLogEntries(t *testing.T) {
 	}
 	testEqual(t, want, res)
 }
+
+func TestIncident_ResponderRequest(t *testing.T) {
+	setup()
+	defer teardown()
+
+	id := "1"
+	mux.HandleFunc("/incidents/"+id+"/responder_requests", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "POST")
+		w.Write([]byte(`{"requester_id": "PL1JMK5", "message": "Help", "responder_request_targets": [{"responder_request_target":{"id":"PJ25ZYX","type":"user_reference"}}]}`))
```

Comment:
This is supposed to match the JSON of the response you're expecting. Can you change it to reflect the response?